app/views/idv/shared/_document_capture.html.erb

```diff
@@ -1,7 +1,8 @@
 <% title t('titles.doc_auth.doc_capture') %>
-<% content_for :meta_tags do %>
+<% content_for :head do %>
   <%= tag.meta name: 'acuant-sdk-initialization-endpoint', content: IdentityConfig.store.acuant_sdk_initialization_endpoint %>
   <%= tag.meta name: 'acuant-sdk-initialization-creds', content: IdentityConfig.store.acuant_sdk_initialization_creds %>
+  <%= stylesheet_link_tag 'document-capture' %>
```

Comment:
This change to render in `<head>` wasn't strictly relevant, but can help avoid [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content).